packages/cfd/src/Containers/migration-banner/__test__/migration-banner.spec.tsx

```diff
@@ -3,6 +3,7 @@ import { screen, render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { Analytics } from '@deriv/analytics';
 import { StoreProvider, mockStore } from '@deriv/stores';
+import { CFDStoreProvider } from '../../../Stores/Modules/CFD/Helpers/useCfdStores';
```

Comment:
**NIT:** Can a testcase to check for a case without an option to upgrade be added (So no banner)?

Reply:
Thanks @likhith-deriv .I have added testcases for the same